internal/pkg/cli/env_init.go

```diff
@@ -257,6 +271,15 @@ func (o *initEnvOpts) RecommendedActions() []string {
 	return nil
 }
 
+// checkExist fetches an application by name. If it can't be found, return a ErrNoSuchApplication
```

Comment:
looks like this method does not return `ErrNoSuchApplication ` this type of error... maybe we can remove this method and move this logic inline since it is pretty straightforward.